packages/admin/admin-stories/src/docs/components/RowActions/stories/RowActions.stories.tsx

```diff
@@ -0,0 +1,82 @@
+import { RowActionsItem, RowActionsMenu } from "@comet/admin";
+import { Accept, Add, BallTriangle, Copy, Delete, Edit, Favorite, FileData, Info, LinkExternal, Online, Paste, Settings } from "@comet/admin-icons";
+import { Divider, Paper } from "@mui/material";
+import { storiesOf } from "@storybook/react";
+import * as React from "react";
+
+storiesOf("stories/components/RowActions/RowActions", module)
+    .addDecorator((Story) => (
+        <Paper
+            variant="outlined"
+            sx={{
+                display: "flex",
+                alignItems: "center",
+                justifyContent: "flex-end",
+                padding: 2,
+                maxWidth: 320,
+                marginLeft: "auto",
+                marginRight: "auto",
+            }}
+        >
+            <Story />
+        </Paper>
+    ))
+    .add("Edit & Context Menu Example", () => {
+        return (
+            <RowActionsMenu>
+                <RowActionsItem icon={<Edit />} componentsProps={{ iconButton: { color: "primary" } }}>
+                    Edit
+                </RowActionsItem>
+                <RowActionsMenu>
+                    <RowActionsItem icon={<Copy />}>Copy</RowActionsItem>
+                    <RowActionsItem icon={<Paste />}>Paste</RowActionsItem>
+                    <RowActionsMenu text="More" icon={<Settings />}>
+                        <RowActionsItem icon={<Online />}>Additional Item 1</RowActionsItem>
+                        <RowActionsItem icon={<Online />}>Additional Item 2</RowActionsItem>
+                        <RowActionsItem icon={<Online />}>Additional Item 3</RowActionsItem>
+                    </RowActionsMenu>
+                    <Divider />
+                    <RowActionsItem icon={<Delete />}>Delete</RowActionsItem>
+                </RowActionsMenu>
+            </RowActionsMenu>
+        );
+    })
+    .add("More Complex & Deeply Nested Example", () => {
```

Comment:
I realized that the nested menus close one after another. So it takes for example three clicks to switch from submenu 2 to submenu 1 if submenu 2 is fully extended (see video). 

https://user-images.githubusercontent.com/13380047/224969784-f03961db-f867-44e6-9f4f-1b0cb0f15f8d.mov


That doesn't seem optimal. Can we improve this?

Reply:
Yes! I believe nested menus shouldn't add an additional backdrop. Clicking away should close the whole menu.

Furthermore, wouldn't it be nice to open/close the nested menu on mouse enter/leave?

Reply:
Yes, it's definitely not optimal.  

I wasn't able to find a easy to implement solution and since nested sub-menus are not currently used that often, I would't prioritize that at the moment. 

Could we ignore this for now? 
Then I'd create a separate Task/MR and discuss this with UX to decide on the optimal behavior, also considering the suggestions from @johnnyomair.

Reply:
Fine by me. Maybe we should remove the nested/complex example for now then?